docs/post-mortems/2019-01-23.md

```diff
@@ -0,0 +1,95 @@
+# January 23rd, 2019 Prow Outage
+
+Created By: Benjamin Elder <bentheelder@google.com>
+Last Modified: 2019-01-23
+
+## Summary
+
+The main Kubernetes production Prow deployment, [prow.k8s.io] stopped responding
+to GitHub events due to the webhook handler ("[hook]") being down.
+
+This resulted in an outage of unknown length bounded to at maximum <compute and fill in>
```

Comment:
This can be the time between first merge config being pushed and the revert config being pushed, since the error was on config load

Reply:
That will be the bound. I didn't compute it yet, ran to grab food. The actual failure should be tighter and only took down hook. I assume hook must have restarted or has a bug. The other components continued to use the prior loaded config.

Reply:
Hmm true, `hook` would have to have crashed or been rescheduled on a node migration/auto-scale

Reply:
what is especially strange is that there are multiple hook pods and they all did this, so I sorta wonder if there's an issue in hook.

Reply:
Interesting. Would love to see the logs from the other pods

Reply:
(before entering the CrashLoopBackOff)